app/views/refinery/pages/home.html.erb

```diff
@@ -30,8 +30,8 @@
           </p>
 
           <div class="hero-banner-actions">
-            <a class="button" href="/goals">Goals</a>
-            <a class="button button--hollow button--with-arrow" href="https://youtu.be/bApghSqlbYc" target="_blank">Watch Video</a>
+            <a class="button" href="/find-out">Find Out</a>
+            <a class="button button--hollow button--with-arrow" href="https://www.youtube.com/playlist?list=PLJNLux-_ZPcINWehD4I2hSsdHQ5V-ttoU" target="_blank">Watch Videos</a>
```

Comment:
This works fine, but if we want to have the first video in the playlist auto-play, we could use this link too: https://www.youtube.com/watch?v=X6fkV1lZITg&list=PLJNLux-_ZPcINWehD4I2hSsdHQ5V-ttoU